### PR TITLE
Adjust ios support

### PR DESCRIPTION
**Pull request by PanicButtonPressed**
### Summary

This PR adjusts the support for ios devices.

---

### Reviewer checklist

*Reviewer agreement:*

* Reviewers assign themselves at the start of the review.
* Reviewers do **not** commit or merge the pull request.
* Reviewers have to check and mark items in the checklist.

**Pull request checklist**

- [x] Conforms to the pull request title naming `AP-XXX <a description in imperative form>`.
- [x] Squash commit conforms to the naming convention `AP-XXX <a description in imperative form>`.
- [x] Linked the ticket in the pull request title or the references section.
- [x] Added an informative pull request summary.

**Code checklist**

- [x] Conforms to the branch naming `AP-XXX-<a_small_stub>`.
- [x] Passed scope checks.
- [x] Added or updated tests if needed.
- [x] Added or updated code documentation if needed.

### CODE REVIEW

File: xain_voice_recognizer/ios/Flutter/.last_build_id [hunk 1]
```diff
@@ -0,0 +1 @@
+e12d2b52d8f4f7aafcd9d4e7160d4899
```
Comment (biernacki) [inline xain_voice_recognizer/ios/Flutter/.last_build_id:1]: Not sure if this file is needed, could you confirm?

File: xain_voice_recognizer/ios/Podfile [hunk 2]
```diff
@@ -87,4 +88,4 @@ post_install do |installer|
       config.build_settings['ENABLE_BITCODE'] = 'NO'
     end
   end
-end
+end
```
Comment (biernacki) [inline xain_voice_recognizer/ios/Podfile:91]: Maybe a empty line at the EOF would be good

Comment (PanicButtonPressed) [reply xain_voice_recognizer/ios/Podfile:91]: Sure, I'll add it.

File: xain_voice_recognizer/ios/Flutter/.last_build_id (hunk 1 above)
Comment (PanicButtonPressed) [reply xain_voice_recognizer/ios/Flutter/.last_build_id:1]: Great catch. Thanks.

**Pull request merged**